File: POLIMORFISMO/EJERCICIO6/videojuego.py
class Videojuego:
    def __init__(self, nombre="Desconocido", plataforma="Desconocida", cantidadDeJugadores=1):
        self.nombre = nombre
        self.plataforma = plataforma
        self.cantidadDeJugadores = cantidadDeJugadores

    def mostrar(self):
        print(f"Nombre: {self.nombre}")
        print(f"Plataforma: {self.plataforma}")
        print(f"Cantidad de jugadores: {self.cantidadDeJugadores}")

    def agregarJugadores(self, cantidad=1):
        self.cantidadDeJugadores += cantidad
        print(f"Se agregaron {cantidad} jugador(es). Total: {self.cantidadDeJugadores}")

v1 = Videojuego()
v1.mostrar()
v1.agregarJugadores()
v1.agregarJugadores(2)

v2 = Videojuego("fifa", "PlayStation")
v2.mostrar()

v3 = Videojuego("Mincraf", "PC", 5)
v3.mostrar()

File: POLIMORFISMO/EJERCICIO6/Videojuego.java
public class Videojuego {
    private String nombre;
    private String plataforma;
    private int cantidadDeJugadores;

    public Videojuego(String nombre, String plataforma, int cantidadDeJugadores) {
        this.nombre = nombre;
        this.plataforma = plataforma;
        this.cantidadDeJugadores = cantidadDeJugadores;
    }

    public Videojuego(String nombre, String plataforma) {
        this.nombre = nombre;
        this.plataforma = plataforma;
        this.cantidadDeJugadores = 1;
    }

    public Videojuego() {
        this.nombre = "Desconocido";
        this.plataforma = "Desconocida";
        this.cantidadDeJugadores = 1;
    }

    public void mostrar() {
        System.out.println("Nombre: " + nombre);
        System.out.println("Plataforma: " + plataforma);
        System.out.println("Cantidad de jugadores: " + cantidadDeJugadores);
    }

    public void agregarJugadores() {
        cantidadDeJugadores += 1;
        System.out.println("Se agregó 1 jugador. Total: " + cantidadDeJugadores);
    }

    public void agregarJugadores(int cantidad) {
        cantidadDeJugadores += cantidad;
        System.out.println("Se agregaron " + cantidad + " jugadores. Total: " + cantidadDeJugadores);
    }
}
